filebeat/docs/how-does-filebeat-work.asciidoc

```diff
@@ -2,68 +2,68 @@
 
 This page intends to explain what the key building blocks of Filebeat are and how they work together. The goal is that the available configuration options can be better understood on what they do.
 
-=== What is a Prospector?
+=== What is an Input?
```

Comment:
From the perspective of working with the config file, "input" is definitely easier to follow and more intuitive.

My concern with renaming "prospector" to "input" is when we start to talk about an input conceptually as a component that does something. We also have the concept of inputs and outputs in Logstash. Our concept of outputs in Beats is pretty similar to Logstash; you specify an output based on the specific type of system that you want to send events to. But in the context of Filebeat, an input does not read from a specific type of system. It's more about specifying the paths to files and defining how you want Filebeat to process the log lines.

When you start explaining things conceptually, I think that "input" starts to sound a bit awkward. The word "input" by itself is almost too generic. It doesn't sound like a component that actually does something. In Logstash, they get around this problem by talking about "input stages" and "input plugins". I think maybe you ran into this problem yourself because you use "input module" below. 

I wonder if we could rename the section without talking about an input as an actual component of functionality within Filebeat.